src/filesystem/File.js

```diff
@@ -28,7 +28,11 @@
 define(function (require, exports, module) {
     "use strict";
     
-    var FileSystemEntry = require("filesystem/FileSystemEntry");
+    var FileSystemEntry = require("filesystem/FileSystemEntry"),
+    Content         = require("extensions/default/brackets-browser-livedev/nohost/src/content"),
+    Filer           = require("filesystem/impls/filer/BracketsFiler"),
```

Comment:
You're not actually using `Filer` here, just `Path`, so you can do `var Path = require("filesystem/impls/filer/BracketsFiler").Path;` and not do this and line 35.